tests/test_dse.py

```diff
@@ -1134,6 +1135,44 @@ def test_full_shape_big_temporaries(self):
         op1(time_M=1)
         assert np.all(u.data == exp)
 
+    @patch("devito.passes.clusters.aliases.MIN_COST_ALIAS", 1)
+    @switchconfig(profiling='advanced')
+    def test_extraction_from_lifted_ispace(self):
+        """
+        Test that the aliases are scheduled correctly when extracted from
+        Clusters whose iteration space is lifted (ie, stamp != 0).
+        """
+        so = 8
+        grid = Grid(shape=(6, 6, 6))
+
+        v = TimeFunction(name="v", grid=grid, space_order=so)
+        v.data_with_halo[:] = 1.
+        p = TimeFunction(name="p", grid=grid, space_order=so, staggered=NODE)
```

Comment:
`staggered=NODE` not needed here?

Reply:
It is needed, all funcs are staggered

Reply:
Why then does the test pass without it?

Reply:
Because it's a dummy equation for the test, but I prefer to have it as it is because for exampel first order acoustic won't work without it.

Reply:
I carved the test out of @speglich 's code, I trimmed down all the unnecessary except for that NODE, which might not make a difference, but won't hurt either. So yeah, this was intentional